vms/platformvm/block/executor/manager.go

```diff
@@ -110,10 +110,8 @@ func (m *manager) NewBlock(blk block.Block) snowman.Block {
 	}
 }
 
-func (m *manager) SetPreference(blkID ids.ID) bool {
```

Comment:
why this change? wouldn't we need to change this back once we introduce the context cancellation?